devicetypes/smartthings/child-switch-health-power.src/child-switch-health-power.groovy

```diff
@@ -0,0 +1,58 @@
+/**
+ *  Copyright 2019 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition(name: "Child Switch Health Power", namespace: "smartthings", author: "SmartThings", mnmn: "SmartThings", vid: "generic-switch-power") {
```

Comment:
Actually, instead of a new DTH here, can you rewrite `child-metering-switch` to be technology-agnostic?

I think this would also require a few changes to the DTH that uses it `zwave-multi-metering-switch`.

Reply:
Would still prefer that you make these changes instead